test/port_forwarding_test.go

```diff
@@ -221,4 +224,79 @@ address=/foobar/1.2.3.4
 			g.Expect(string(reply)).To(Equal("OK"))
 		}).Should(Succeed())
 	})
+
+	// unsure of if AF_UNIX socket support in windows just works this way; but windows seems to support AF_UNIX sockets
+	if runtime.GOOS != "windows" {
```

Comment:
Yes good point. 

You can write it even more nicely with:

```
It("should do something, if it can", func() {
  if runtime.GOOS != "windows" {
    Skip("AF_UNIX not supported on Windows")
  }
  ...
})


```

With this, we have the same number of tests on all platforms and a clear indication on Windows: this test is skipped and why.


(this is really a nice to have, I never ran the test suite on Windows)

Reply:
Might also help if I compare it right. It should have been `==`.

This is done. I noticed last night the CI was failing at tests, but the tests have been working locally. Not sure what's up with that.

```
•time="2022-01-06T09:20:41-06:00" level=error msg="accept tcp [::]:9090: use of closed network connection"
•••••
Ran 17 of 17 Specs in 52.122 seconds
SUCCESS! -- 17 Passed | 0 Failed | 0 Pending | 0 Skipped
--- PASS: TestSuite (52.12s)
PASS
ok  	github.com/containers/gvisor-tap-vsock/test	52.281s
```

Reply:
Great!

I merged a "fix" for the CI. The macOS worker is really slow.. I reduced the load on it by avoiding to pull large docker images for test with just a small binary launched in the VM (which embeds a test dns server and http server).

Reply:
👍 

Going to mark this resolved since the worker issue should probably be considered separate from writing the test case then.